core/services/helpers/DecimalValues.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace EventEspresso\core\services\helpers;
+
+use EE_Currency_Config;
+
+class DecimalValues
```

Comment:
Very nice. Needed those. Those methods will be of help.

Reply:
thnx. I might change that class name as I don't like it very much.
Was avoiding using `DecimalUtilities` because it's harder to type without making a stupid typo